src/schema/v2/user.ts

```diff
@@ -79,6 +117,29 @@ export const UserType = new GraphQLObjectType<any, ResolverContext>({
       description: "The given phone number of the user.",
       type: GraphQLString,
     },
+    createdAt: date(({ created_at }) => created_at),
+    emailConfirmedAt: date(({ email_confirmed_at }) => email_confirmed_at),
+    secondFactorEnabled: {
+      description:
+        "If the user has enabled two-factor authentication on their account",
+      type: GraphQLBoolean,
+      resolve: ({ second_factor_enabled }) => second_factor_enabled,
+    },
+    roles: {
+      description: "The roles of the user",
+      type: new GraphQLList(GraphQLString),
+      resolve: ({ roles }) => roles,
```

Comment:
Note that there's no need to resolve something if the graphql field name is identical (and doesn't need to be camelCased)